docs/manual/projects/node-execution/ssh.md

```diff
@@ -1,29 +1,39 @@
 # SSH Node Execution
 
-Rundeck by default uses SSH to execute commands on remote nodes, SCP to copy scripts to remote nodes, and locally executes commands and scripts for the local (server) node.
+Rundeck employs SSH as its default protocol for remote command execution and SCP for script file transfers to remote nodes. For local (server) nodes, commands and scripts are executed directly on the system.
 
-The SSH plugin expects each node definition to have the following properties in order to create the SSH connection:
+## Core Requirements
+
+The SSH plugin requires specific node configuration parameters for establishing secure connections:
```

Comment:
Do we want to specify that we recommend using the SSHJ plugin? This is the default Node Executor for new projects, since it supports the latest encryption algorithms.

Reply:
I added a blurb on it.  I wonder what it would take to change the default in the product and if it would impact existing setups?

Reply:
i mean, we already made SSHJ the default. All new projects created have that set as the Default Node Executor

Reply:
OK.  I checked some of mine and they were old, but I'll try a new one.